### DISCUSSION TITLE
Upgrading from Monaco Language Client 4.0.3 to 5.0.0

### DISCUSSION THREAD

**Discussion by Taco7** [TypeFox/monaco-languageclient#598] [Q&A]
I am upgrading an Angular project where the existing monaco-language client is 4.0.3 to 5.0.0
There are no compilation issues in the code I am running into this error message `/node_modules/monaco-languageclient/lib/monaco-services.js:7:0-43 - Error: Module not found: Error: Can't resolve 'vscode/services' in '/Users/adityaverma/Documents/texera/core/new-gui/node_modules/monaco-languageclient/lib'`

Is there something that I am missing for this upgrade?

**Comment by kaisalmen** [comment]
Hi @Taco7 this looks like a missing or wrong dependency, Any particular reason you upgrade from 4.0.3 to 5.0.0 and not the latest version. Are aware we have an angular example repository: https://github.com/TypeFox/monaco-languageclient-ng-example
